apps/antalmanac/package.json

```diff
@@ -3,21 +3,25 @@
     "version": "1.0.0",
     "private": true,
     "homepage": "https://antalmanac.com",
-    "description": "A course exploration and scheduling tool for UCI Anteaters",
+    "description": "the premiere course exploration and planning tool for UCI anteaters",
```

Comment:
Why no caps 🤔

Reply:
Idk, I just looked through a lot of package.jsons and felt that they used really informal punctuation. Like no capitalization or period. I'd do both so it's formal, but idk :shrug: